app/src/main/java/org/wikipedia/page/PageActivity.kt

```diff
@@ -495,6 +497,18 @@ class PageActivity : BaseActivity(), PageFragment.Callback, LinkPreviewDialog.Ca
                 ActivityCompat.getReferrer(this)?.let { uri ->
                     historyEntry.referrer = uri.toString()
                 }
+                // TODO: update this after we have the finalized URL format.
```

Comment:
can we remove the todos here since url format is finalized?

Reply:
yep